packages/viz/src/lib/sources/DOSource.ts

```diff
@@ -126,6 +129,13 @@ export class DOSource extends Source {
     this._variable = variable;
 
     this._baseURL = `${this._credentials.serverURL}api/v4/data/observatory`;
+
+    this._CDNURL = `https://do-gusc-{domain}.cartocdn.com/USER/api/v4/data/observatory/`;
```

Comment:
Given that this variable is constant we can define it while declaring the variable above. And I guess that we should replace that `USER` in the URL with something else.

Reply:
This issue has been reported to the backend team, but it right now requires to pass the username as a query parameter. Once they'll fix it, we'll replace it.